CHANGELOG.md

```diff
@@ -12,6 +12,12 @@ changes.
 
 - API server responses now contain a `timestamp` and a monotonic `seq`uence number. [#618](https://github.com/input-output-hk/hydra/pull/618)
 
+- **BREAKING** Change the way the internal wallet initializes its state [#621](https://github.com/input-output-hk/hydra/pull/621)
```

Comment:
I don't see how this would be a breaking change. Can you explain? Is it related to state storage maybe?

Reply:
Yeah.. maybe too strict to call changes of the log spec a breaking change.

Reply:
OK, no problem. Makes sense but I would just make it explicit that these are the logs that are _brokenly changed_.

Reply:
```suggestion
- **BREAKING logs** Change the way the internal wallet initializes its state [#621](https://github.com/input-output-hk/hydra/pull/621)
```